tests/fixtures/__init__.py

```diff
@@ -118,13 +105,25 @@ async def fixture_container(
         **mongodb_fixture.config.dict(),
     )
 
+    # publish an event announcing a dataset
     async with get_container(config=config) as container:
-        await publish_datasets(container)
+        await kafka_fixture.publisher.publish(
```

Comment:
The KafkaFixture even has this syntactic sugar:
```suggestion
        await kafka_fixture.publish_event(
```